treeview/js/main.js

```diff
@@ -558,6 +558,12 @@
                 function getTreeData(queryConfig) {
                     var treeHeaders = {};
                     treeHeaders[ERMrest.contextHeaderName] = getHeader("main", queryConfig.tree_schema_table);
+                    if(urlParams['pcid'])
+                        header['pcid']=urlParams['pcid']
+                    if(urlParams['ppid'])
+                        header['ppid']=urlParams['ppid']
+                    treeHeaders[ERMrest.contextHeaderName] =header;
+                    treeHeaders[ERMrest.contextHeaderName] = ERMrest._certifyContextHeader(treeHeaders[ERMrest.contextHeaderName]);
```

Comment:
Header is being set to 2 values above when header is not defined.

From my comment before:
```
function getTreeData(queryConfig) {
  var treeHeaders = {};
  var header = getHeader("main", queryConfig.tree_schema_table);
  if(urlParams['pcid'])
    header['pcid']=urlParams['pcid']
  if(urlParams['ppid'])
    header['ppid']=urlParams['ppid']

  treeHeaders[ERMrest.contextHeaderName] = ERMrest._certifyContextHeader(header);
```

Reply:
I think I misunderstood your comment. Made the changes